src/clustercoeffs.jl

```diff
@@ -39,16 +38,16 @@ function clustercoeffs(A::MatrixNetwork,weighted::Bool,normalized::Bool)
     end
     
     (rp,ci,ai) = (A.rp,A.ci,A.vals)
-    if length(find(ai.<0)) != 0
+    if length(findall(ai.<0)) != 0
```

Comment:
Yes. This can become this `typeof(findfirst(x.<0)) != Nothing` (that's because findfirst can return a number or _nothing_ and that is of type Nothing)

Reply:
This 9a6ee92c2acef39d24ad2cd37a2625fe3841222b fixes it

Reply:
Typo fix `x` -> `ai` e78c50bb1ca0413f5207c6f904ee066672d4ee99